src/WebJobs.Extensions.DurableTask/DurableOrchestrationClientBase.cs

```diff
@@ -12,6 +12,11 @@ namespace Microsoft.Azure.WebJobs
     /// </summary>
     public abstract class DurableOrchestrationClientBase
     {
+        /// <summary>
+        /// Provides Task Hub's name
```

Comment:
For consistency with other getter-properties, let's use the text "Gets the name of the task hub configured on this client instance."